lib/i18n.rb

```diff
@@ -261,14 +261,14 @@ def exists?(key, _locale = nil, locale: _locale, **options)
     #
     # Setting a Hash using Ruby:
     #
-    #     store_translations(:de, :i18n => {
-    #       :transliterate => {
-    #         :rule => {
-    #           "ü" => "ue",
-    #           "ö" => "oe"
-    #         }
-    #       }
-    #     )
+    #     store_translations(:de, i18n: {
+    #                          transliterate: {
+    #                            rule: {
+    #                              'ü' => 'ue',
+    #                              'ö' => 'oe'
+    #                            }
```

Comment:
Not sure why rubocop sees this hash with '=>' syntax as valid.

Reply:
As valid or invalid?

Reply:
@radar as **valid**, as far as I understand, rubocop should shout at me to change this but it actually does not and maybe it's because of the nestetation?

Reply:
@radar so WDYT, should we stick to the rubocop convention and add something like this:
```ruby
     store_translations(:de, i18n: {
                          transliterate: {
                            rule: {
                              'ü': 'ue',
                              'ö': 'oe'
                            }
                          }
                        })
```
or maybe stick to the existing convention like that:
```ruby
    #     store_translations(:de, :i18n => {
    #       :transliterate => {
    #         :rule => {
    #           "ü" => "ue",
    #           "ö" => "oe"
    #         }
    #       }
    #     })
```

Reply:
I'm guessing Rubocop doesn't mind if it's a string or a symbol in this case because it's not an ascii character. I don't know the exact reason, and I don't have a preference for one over the other.